corehq/apps/domain/models.py

```diff
@@ -994,7 +994,7 @@ def transfer_domain(self, by_user, *args, transfer_via=None, **kwargs):
         if by_user:
             log_user_change(self.domain, couch_user=self.from_user,
                             changed_by_user=by_user, changed_via=transfer_via,
-                            message=_("Removed from domain '{domain_name}'").format(domain_name=self.domain))
+                            message=f"Removed from domain '{self.domain}'")
```

Comment:
newly added this week
https://github.com/dimagi/commcare-hq/commit/c7e498725df7a04c5b8133fbbaa750c6fbe3486b#diff-c9557c19413e04ae10a25832b30f73ee68d4e9bdae31b49f5404b9086b80f126R14386